Sheet 2/ProblemF.cpp

```diff
@@ -0,0 +1,29 @@
+#include <iostream>
+#include <cmath>
+
+using namespace std;
+
+int main()
+{
+    int a, b, d = 0;
```

Comment:
Probably better to use `unsigned int`.

Although it won't matter because A & B <=10^9, so the largest A & B in binary are `0011 1011 1001 1010 1100 1010 0000 0000`, it would matter if A & B could be any unsigned 32-bit int.